src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -44,13 +46,24 @@ def acr_import(cmd,
 
     if source_registry:
         if is_valid_resource_id(source_registry):
+            registry, _ = get_registry_by_name(cmd.cli_ctx, parse_resource_id(source_registry)["name"])
             source = ImportSource(resource_id=source_registry, source_image=source_image)
+
         else:
             registry = get_registry_from_name_or_login_server(cmd.cli_ctx, source_registry, source_registry)
             if registry:
                 source = ImportSource(resource_id=registry.id, source_image=source_image)
             else:
                 raise CLIError(SOURCE_REGISTRY_NOT_FOUND)
+
+        try:
+            if registry.login_server.lower() in source_image.lower():
+                logger.warning("\nImporting image '%s/%s'...\nPlease ensure that '--source' is a source image and "
```

Comment:
Is it possible to only warn after the command fails?  This could be noise to the user if they actually want to do this.

Reply:
It should be possible.... I will look into this

Reply:
FYI: @nathana1 this only warns the user if the login server is in the source image. It doesn't necessarily warn the user if thy use both options...

Reply:
@nathana1 made this change.... code is more complex but users should only be warned if appropriate.... I tested with an image that has loginserver in it.

Here is a screenshot of behavior now
![acr import warning](https://user-images.githubusercontent.com/15149427/67529582-6a7fab80-f671-11e9-9eb8-76ed48a6f713.PNG)